pkg/injector/monitoring/metrics.go

```diff
@@ -0,0 +1,85 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package monitoring
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	appID   = "app_id"
+	podName = "pod_name"
+)
+
+var (
+	successfulAdmissionReviewResponseTotal = stats.Int64(
+		"injector/successful_admissionreview_response_total",
+		"The total number of successful admission review responses.",
+		stats.UnitDimensionless)
+	failedAdmissionReviewResponseTotal = stats.Int64(
+		"injector/failed_admissionreview_response_total",
+		"The total number of failed admission review responses.",
+		stats.UnitDimensionless)
+
+	// appIDKey is a tag key for App ID
+	appIDKey = tag.MustNewKey(appID)
+
+	// podNameKey is a tag key for Pod name
+	podNameKey = tag.MustNewKey(podName)
+)
+
+// RecordSuccessfulAdmissionReviewResponseCount records the number of successful admission review responses
+func RecordSuccessfulAdmissionReviewResponseCount(appID, podname string) {
+	stats.RecordWithTags(context.Background(), withTags(appIDKey, appID, podNameKey, podname), successfulAdmissionReviewResponseTotal.M(1))
+}
+
+// RecordFailedAdmissionReviewResponseCount records the number of failed admission review responses
+func RecordFailedAdmissionReviewResponseCount(appID, podname string) {
```

Comment:
Can we track `failedReasonKey` tag key to track the error reason ? Then we can pass simple short failure reason. it allows operator to know the failure reason without seeing the log. 

for example,
https://github.com/dapr/dapr/blob/034b82ebfbb7c76e5801ae6d90c2edd29f629df8/pkg/diagnostics/service_monitoring.go#L229